test/e2e/cli_test.go

```diff
@@ -1332,6 +1332,47 @@ func (s *CLISuite) TestWorkflowTemplateRefSubmit() {
 	})
 }
 
+func (s *CLISuite) TestWorkflowCopyArtifact() {
+	s.Given().
+		Workflow("@testdata/artifact-repository-ref.yaml").
```

Comment:
I don't think you should re-use this workflow, use `basic-workflow.yaml` instead, that is less likely to change. Or create your own for the test.

Reply:
Added in a workflow (`basic-artifact-workflow.yaml`) specifically for the cp e2e test.